examples/webgl_loader_viewer.html

```diff
@@ -0,0 +1,410 @@
+		// http://stackoverflow.com/questions/11766163/smart-centering-and-scaling-after-model-import-in-three-js
+		// fit camera to object
+		var bBox = new THREE.Box3().setFromObject(scene);
+		var height = bBox.size().y;
+		var dist = height / (2 * Math.tan(camera.fov * Math.PI / 360));
+		var pos = scene.position;
+
+		// fudge factor so the object doesn't take up the whole view
+		camera.position.set(pos.x, pos.y, dist * 3);
+		camera.lookAt(pos);
+	}
+
+	function toggleDiv(id) {
+		var div = document.getElementById(id);
+		div.style.display = (div.style.display === 'block') ? 'none' : 'block';
+	}
+
+	function clearDiv(id) {
+		var div = document.getElementById(id);
+		div.innerHTML = "";
+	}
+
+	function appendDiv(id, extra) {
+		var div = document.getElementById(id);
+		div.innerHTML = div.innerHTML + extra;
+	}
+
+	function debugPrint(text)
+	{
+		appendDiv('debug',text);
+	}
+
+	function clearScene() {
+		clearDiv('debug');
+		camera.position.z = 250;
+		scene.scale.x = scene.scale.y = scene.scale.z = 1.0;
+
+		for( var i = scene.children.length - 1; i >= 0; i--){
+			obj = scene.children[i];
+			scene.remove(obj);
+		}
+	}
+
+	function setupLights() {
+		var ambient = new THREE.AmbientLight( 0x101030 );
+		scene.add( ambient );
+
+		var directionalLight = new THREE.DirectionalLight( 0xffeedd );
+		directionalLight.position.set( 0, 0, 1 );
+		scene.add( directionalLight );
+	}
+
+	function reloadModel(objPath)
+	{
+		controls.reset();
+		loadModel(objPath);
+	}
+
+	function loadModel(objPath)
+	{
+		var token = objPath.split('.');
+		var fileType = token[token.length-1];
+
+		if (fileType === "json")
+		{
+			if (token[token.length-2] === "assimp")
+				fileType = "assimp.json";
+ 		}
+
+		token = objPath.split('/');
+		var fileName = token[token.length-1];
+
+		var onProgress = function ( xhr ) {
+			if ( xhr.lengthComputable ) {
+				var percentComplete = xhr.loaded / xhr.total * 100;
+				console.log( Math.round(percentComplete, 2) + '% downloaded');
+			}
+		};
+
+		var onError = function ( xhr ) {
+		};
+
+		var manager = new THREE.LoadingManager();
+		manager.onProgress = function ( item, loaded, total ) {
+
+			console.log( item, loaded, total );
+
+		};
+
+		var loader = selectLoader(fileType, manager);
+
+		if (fileType === "dae" )
+		{
+			loader.options.convertUpAxis = true;
+		}
+		else if (fileType === "pcd"){
+			//do something to invert upside down
+		}
+
+		clearScene();
+
+		//to do: improvement - setupLights() improve to use better lights
+		setupLights();
+
+		//to do: improvement - loader.load() note: sea loader does not use onload function - loader.load 2nd param
+		loader.load( objPath , function ( myScene ) {
+
+			debugPrint(objPath+"<br>");
+			OnLoadScene(myScene, fileType, fileName);
+			resetCamera();
+
+		}, onProgress, onError );
+
+		//to do: improvement - post loader.load() fixes - handle camera for these files automaticaly
+		if (fileName==="mascot.tjs.sea")
+		{
+			camera.position.z = 2000;
```

Comment:
I fixed the camera of .tjs.sea model by using a custom param with the SEA loader, as SEA has its own node adding to scene mechanism and seems to ignore the camera settings otherwise.